plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
I have changed the word "global" into "common". While this does not fix the entire philosophical issue of not specifying the context of the word global/general/common, it does take away the confusion over the term "global" which has such a strong connotation in Cura.